ghcide.cabal

```diff
@@ -340,6 +340,7 @@ test-suite ghcide-tests
         process,
         QuickCheck,
         quickcheck-instances,
+        record-dot-preprocessor,
```

Comment:
Not sure this builds on all the platforms we test on

Reply:
Is there a bulitin (or working) plugin that uses `parsedResultAction`?

Reply:
I think this might work actually because we are ditching 8.4 support. #803